### ISSUE TITLE
Varos Cloudstrider - Azure Ring Dragon Beam

### ISSUE THREAD

**Issue by Godmatik** [gamefreedomgit/Frostmourne#724]
The Beam during the Varos fight always follows the person with highest threat, which always ends up being the healer.

It should target and track a random non-tank target.

**Comment by UltraNix** [comment]
Just tested - targets random players.

**Comment by Godmatik** [comment]
> Just tested - targets random players.



> Just tested - targets random players.

Let me rephrase - it targets a random player, but then follows an aggro table, and immediately goes for the healer

**Comment by UltraNix** [comment]
Okay.

**Comment by UltraNix** [comment]
Fixed.
